services.html

```diff
@@ -1,38 +1,230 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="utf-8">
-	<title>Services</title>
-	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
-	<link rel="stylesheet" href="css/index.css">
-	<!--[if IE]>
-		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-	<![endif]-->
-</head>
+  <head>
+    <meta charset="utf-8" />
+    <title>Services</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,700"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
+    <!--[if IE]>
+      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+  </head>
 
-<body>
+  <body>
+    <div class="services-container">
+      <section class="services-menu-content">
+        <h2>S&J</h2>
+        <nav>
+          <a href="/index.html">home</a> <a href="#">contact</a>
+          <a href="#">about</a>
+        </nav>
+      </section>
 
-	<h1>Services</h1>
+      <header class="bind-services">
+        <img src="/img/services/services-jumbotron.png" alt="" />
```

Comment:
Again uneccessary slashes `/`, this time in services file. Because of wrong paths images do not display and links do not work.